apps/web/playwright/test-min-max-characters.e2e.ts

```diff
@@ -0,0 +1,158 @@
+import { expect } from "@playwright/test";
```

Comment:
@mhetreayush We  have playwright/manage-booking-questions.e2e.ts for form builder related tests.
We should move this test there. 

Also, could you see if we can test `textarea` field along with any existing test in there? If not that's also fine. The tests there follow the same flow, it is just about adding and verifying a new field.

I wanted to ensure that we don't introduce new e2es when when we can amend any existing e2e. As an e2e test is time consuming.

Reply:
Will do that.

Reply:
Also, maybe we could be fine adding unit test for `textarea` schema in schema.ts.
And, supportsLengthCheck related component code can also be unit tested with react-testing-library. For that you would have to refactor supportsLengthCheck related code into a new component(which really makes sense anyway)